doc/user/content/quickstarts/streaming.md

```diff
@@ -0,0 +1,202 @@
+
+Reuse your `psql` session and build the analytics:
+
+1. Create a [`MATERIALIZED VIEW`](/sql/create-materialized-view/) to aggregate the purchase prices for vip users that haver not been refunded.
+
+    A `MATERIALIZED VIEW` is persisted in a durable storage and is incrementally updated as new data arrives.
+
+    ```sql
+    CREATE MATERIALIZED VIEW vip_purchases AS
+        SELECT
+            is_vip,
+            SUM(purchase_price) as total_purchase_price
+        FROM analytics.dbt_dennis.purchases
+        LEFT JOIN analytics.dbt_dennis.users
+            ON purchases.user_id = users.id
+        WHERE deleted is false
+        GROUP BY 1;
+    ```
+
+    The above model will update as new purchases come in and if existing purchases are marked as deleted later (soft deletes).
+
+### Subscribe to updates
+
+`SUBSCRIBE` can stream updates from materialized views as they occur. Use it to verify how the analytics change over time.
+
+1. Subscribe to the vip purchases:
+    ```sql
+    COPY (SUBSCRIBE (SELECT * FROM shop.vip_purchases)) TO STDOUT;
+    ```
+
+    As soon as we subscribe we will see the current state of our materialized view.
+    ```
+    1671724467569	1	f	35
+    1671724467569	1	t	25
+    ```
+
+    If there is a new purchase or a soft delete, the subscription will show two new rows. The previous value and the new value. In this case there was a soft delete of a non vip purchase of $15.
+    ```
+    1671724467569	1	f	35
+    1671724467569	1	t	25
+    1671724514737	1	f	20
+    1671724514737	-1	f	35
+    ```
+
+1. Press `CTRL + C` to interrupt the subscription after a few changes.
+
+### Connect to BI Tool
+
+Instead of using the subscribe we are will connect our BI tool directly on top of our materialized view. For this example we will use [Metabase](https://www.metabase.com/). Though since Materialize is wire-compatible with PostgreSQL, any tool that supports PostgreSQL should work.
+
+1. In Metabase add a new database connection
+
+    | Connection Field | Value |
+    | --- | --- |
+    | Database type | PostgreSQL |
+    | Display Name | Materialize |
+    | Host | `MATERIALIZE_HOST` |
+    | Port | 6875 |
+    | Database name | `materialize` |
+    | Username | `MATERIALIZE_USERNAME` |
+    | Password | `MATERIALIZE_PASSWORD` |
+    | Use a secure connection (SSL) | True |
+
+2. To make a quick dashboard for our mmaterialized view, click "Browse Data" on the left side. Then drill down until you find the `vip_purchases` view. By default it will just be a table but we can change this to a "Bar" chart and get live udpates of new purchases as they are created or deleted.
+
+
+## Recap
+
+In this quickstart, you saw:
+
+-   How to define sources and materialized views within Materialize
+-   How to subscribe to materialized views
+-   Connecting a BI tool to a materialized view
+
+## Related pages
```

Comment:
Instead of "Related pages", for quickstarts we should have a "Lean more" section with some CTA (not sure to what yet).